cmd/gaia/testnets/README.md

```diff
@@ -88,194 +99,200 @@ Your full node has been cleanly upgraded!
+  * e.g. `cosmosaccaddr15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
+
+- `cosmosaccpub`
+  * Derived from account keys generated by `gaiacli keys add`
+  * e.g. `cosmosaccpub1zcjduc3q7fu03jnlu2xpl75s2nkt7krm6grh4cc5aqth73v0zwmea25wj2hsqhlqzm`
+
+- `cosmosvalpub`
+  * Generated when the node is created with `gaiad init`. Needed when staking.
+  * e.g. `cosmosvalpub1zcjduc3qcyj09qc03elte23zwshdx92jm6ce88fgc90rtqhjx8v0608qh5ssp0w94c`
+
+### Key Generation
 
 You'll need a private and public key pair \(a.k.a. `sk, pk` respectively\) to be able to receive funds, send txs, bond tx, etc.
 
 To generate a new key \(default _ed25519_ elliptic curve\):
 
-```
-gaiacli keys add <your_key_name>
+```bash
+gaiacli keys add <key_name>
 ```
 
-Next, you will have to create a passphrase. Save the _seed_ _phrase_ in a safe place in case you forget the password.
+Next, you will have to create a passphrase to protect the key on disk. The output of the above command will contain a _seed phrase_. Save the _seed phrase_ in a safe place in case you forget the password!
 
-If you check your private keys, you'll now see `<your_key_name>`:
+If you check your private keys, you'll now see `<key_name>`:
 
-```
-gaiacli keys show <your_key_name>
+```bash
+gaiacli keys show <key_name>
 ```
 
 You can see all your available keys by typing:
 
-```
+```bash
 gaiacli keys list
 ```
 
 View the validator pubkey for your node by typing:
 
-```
+```bash
 gaiad tendermint show_validator
 ```
 
-Save your address and pubkey to environment variables for later use:
-
-```
-MYADDR=<your_newly_generated_address>
-MYPUBKEY=<your_newly_generated_public_key>
-```
-
 **WARNING:** We strongly recommend NOT using the same passphrase for multiple keys. The Tendermint team and the Interchain Foundation will not be responsible for the loss of funds.
 
-## Get Tokens
-
-The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org).
-
-After receiving tokens to your address, you can view your account's balance by typing: 
-
-```
-gaiacli account <your_newly_generated_address>
-```
-
-Note: When you query an account balance with zero tokens, you will get this error: `No account with address <your_newly_generated_address> was found in the state.` This is expected! We're working on improving our error messages.
-
-## Send Tokens
-
-```
-gaiacli send --amount=10faucetToken --chain-id=<name_of_testnet_chain> --name=<key_name> --to=<destination_address>
-```
+## Fund your account
 
-Note: The `--amount` flag accepts the format `--amount=<value|coin_name>`.
+The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org). The facuet needs the `cosmosaccaddr` from the key you wish to use for staking.
```

Comment:
this kind of ties in to what i mentioned about about using keys vs accounts. also, small typo.

"The `faucet` needs the cosmosaccaddr from the `account` you wish to use for staking."